controllers/mealsController.js

```diff
@@ -1,29 +1,79 @@
 const Meal = require('../models').Meal
 const User = require('../models').User
+const Food = require('../models').Food
+const MealFood = require('../models').MealFood
 const pry = require('pryjs')
 
 const create = (req, res) => {
```

Comment:
Weird, I had a problem using the word create for a route name. Did you hit any errors with using create for your route?

Reply:
Nope! It worked fine for me. Shrug.